inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -163,6 +163,13 @@ def pdf_validator(form, field):
     if field.data and not is_pdf_link(field.data):
         raise StopValidation(message)
 
+def year_validator(form, field):
```

Comment:
Try to write a unit test for your validator, patterning it on the examples in https://github.com/inspirehep/inspire-next/blob/master/tests/unit/forms/test_forms_validators_simple_fields.py.